packages/gnome_mines.rb

```diff
@@ -1,6 +1,6 @@
-require 'package'
+require 'buildsystems/meson'
 
-class Gnome_mines < Package
+class Gnome_mines < Meson
   description 'GNOME Mines (formerly known as Gnomine) is minesweeper clone for GNOME'
   homepage 'https://wiki.gnome.org/Apps/Mines'
   @_ver = '3.36.1'
```

Comment:
Update to [40.1](https://download.gnome.org/sources/gnome-mines/40/gnome-mines-40.1.tar.xz)?